icgcget/cli.py

```diff
@@ -18,78 +20,84 @@
 
 import logging
 import os
+import sys
 import json
 import click
 
-from commands.versions import versions_command
-from commands.reports import StatusScreenDispatcher
-from commands.download import DownloadDispatcher
-from commands.access_checks import AccessCheckDispatcher
-from commands.utils import compare_ids, config_parse, validate_ids, load_json
-from commands.configure import ConfigureDispatcher
+from icgcget.clients.version import __version__
+from icgcget.clients.click_repo import RepoParamType, Logfile
```

Comment:
Let's keep the naming consistent. `RepoParam`, `LogfileParam` or `Repo`, `Logfile`